README.md

```diff
@@ -25,12 +25,24 @@ In a windows command prompt:
     cd src/ManageCourses.Api
     dotnet run
 
+## Database
+
+There's a script for setting up a postgres user and database: `setup-pg-user.sh`
+
 ## Config
 
 An example of the config keys that are required for Secret Manager are available from:
 
 	src\ManageCourses.Api\appsettings.SecretManager_Example.json
 
+E.g.
+
+    dotnet user-secrets set MANAGE_COURSES_POSTGRESQL_SERVICE_HOST localhost
+    dotnet user-secrets set PG_DATABASE manage
+    dotnet user-secrets set PG_USERNAME manageusr
+    dotnet user-secrets set PG_PASSWORD manageusr
```

Comment:
It is generally a bad practice to state a `password`  and other values as that become a given default to use.

I suggest you change it 
dotnet user-secrets set PG_PASSWORD `Your_password`